torchao/testing/model_architectures.py

```diff
@@ -177,3 +178,63 @@ def create_model_and_input_data(
     else:
         raise ValueError(f"Unknown model type: {model_type}")
     return model, input_data
+
+
+class Experts(nn.Module):
```

Comment:
maybe call it something like `FeedForwardWithExperts`? `Experts` is ambiguous

Reply:
this is adapted from https://github.com/meta-llama/llama-models/blob/a9c89c471f793423afd4cc3ca8671d6e56fe64cb/models/llama4/moe.py#L22, how about renaming to `LLama4Experts` to make it more specific